linting/workflows/pr_validation_caller.yml

```diff
@@ -1,29 +1,48 @@
+# =========================================================================================
+# CAMARA Project - Pull Request Validation Workflow Caller
+#
+# This GitHub Actions workflow is responsible for invoking a reusable PR validation workflow
+# from the camaraproject/tooling repository. It is intended to ensure consistent validation
+# steps for all PRs targeting the main branch in this repository.
+#
+# CHANGELOG:
+# - 2025-07-25: Initial version
+#
+# USAGE:
+# - Automatically triggers on pull requests targeting main.
+# - Can be triggered manually via workflow_dispatch.
+# - Calls by default the reusable workflow at 
+#   camaraproject/tooling/.github/workflows/pr_validation.yml@main.
+#
+# DOCUMENTATION:
+# see https://github.com/camaraproject/tooling/tree/main/linting/docs
+# =========================================================================================
+
 name: Caller for PR validation workflow
-# CAMARA Project - Github Action for Pull Reqests
-# This workflow calls the reusable workflow from the tooling repository
-# initial version
 
 on:
-  # Pull Requests to main
+  # Trigger on pull requests to the main branch only
   pull_request:
     branches: main
-  # manual workflow triggering
+  # Enable manual trigger via the GitHub UI
   workflow_dispatch:
   
 concurrency:
   group: ${{ github.ref }}-${{ github.workflow }}
   cancel-in-progress: true
 
 permissions:
+  # Grant necessary write permissions for PRs, contents, and issues
   pull-requests: write
   contents: write
   issues: write
 
 jobs:
   pr_validation:
+    # Invoke the reusable PR validation workflow from the main branch of camaraproject/tooling
     uses: camaraproject/tooling/.github/workflows/pr_validation.yml@main
     secrets: inherit
-# This workflow calls the reusable workflow from tooling repository
-# Tools configuration from the tooling repository branch indicated by `configurations` variable    
+# Tools configuration from the tooling repository branch indicated by `configurations` variable
+# If needed, you can specify a configuration from another branch of camaraproject/tooling (uncomment below)
 #    with:
 #      configurations: staging
```

Comment:
I would rather select the branch already within line 43 in the uses statement, instead of mixing the workflow from one branch with configuration from another branch.

So using workflow *and* configuration from "staging" branch would be 

`   uses: camaraproject/tooling/.github/workflows/pr_validation.yml@staging`

Further (API Repository specific) configuration I would read in future from a configuration file within the API Repository, leaving the workflow untouched.